parlai/tasks/bot_adversarial_dialog/README.md

```diff
@@ -0,0 +1,16 @@
+Task: Bot-Adversarial Dialogue Dataset
+===========================
+Description: Dialogue datasets labeled with offensiveness from Bot-Adversarial Dialogue task
```

Comment:
Can you add a placeholder here for adding the arxiv link for when the paper appears on arxiv? Can you also link to the projects folder?